build.gradle.kts

```diff
@@ -863,11 +865,11 @@ tasks {
             ":idea:idea-fir:test",
             ":idea:idea-frontend-api:test",
             ":idea:idea-frontend-fir:test",
-            ":idea:idea-frontend-fir:idea-fir-low-level-api:test"
+            ":idea:idea-frontend-fir:idea-fir-low-level-api:test",
+            ":plugins:uast-kotlin-fir:test"
```

Comment:
I put this module on `idea-fir-plugin-tests` as well as `miscIdeTests` below. Let me know which one you prefer.

Reply:
@darthorimar any inputs? Is it okay to put newly added tests to both tasks?

Reply:
adding `:plugins:uast-kotlin-fir:test` `to idea-fir-plugin-tests` and to `miscIdeTests` means that FIR UAST test will be run twice on TeamCity. Let's keep it only in `idea-fir-plugin-tests`

Reply:
Thanks for the confirmation. Added to `idea-fir-plugin-tests` only.